src/pathfinder/yapf/nodelist.hpp

```diff
@@ -24,13 +23,12 @@ class CNodeList_HashTableT {
 public:
 	typedef Titem_ Titem;                                        ///< Make #Titem_ visible from outside of class.
 	typedef typename Titem_::Key Key;                            ///< Make Titem_::Key a property of this class.
-	typedef SmallArray<Titem_, 65536, 256> CItemArray;           ///< Type that we will use as item container.
 	typedef CHashTableT<Titem_, Thash_bits_open_  > COpenList;   ///< How pointers to open nodes will be stored.
 	typedef CHashTableT<Titem_, Thash_bits_closed_> CClosedList; ///< How pointers to closed nodes will be stored.
 	typedef CBinaryHeapT<Titem_> CPriorityQueue;                 ///< How the priority queue will be managed.
 
 protected:
-	CItemArray      m_arr;        ///< Here we store full item data (Titem_).
+	std::vector<Titem_> m_arr;    ///< Here we store full item data (Titem_).
```

Comment:
Probably the same here. std::deque has stable pointer-to-item. std::vector does not.